docs/basic-features/image-optimization.md

```diff
@@ -11,199 +11,173 @@ description: Next.js supports built-in image optimization, as well as third part
 
-The following describes the caching algorithm for the default [loader](#loader). For all other loaders, please refer to your cloud provider's documentation.
+Because `next/image` is designed to guarantee good performance results, it cannot be used in a way that will contribute to layout shift, and **must** be sized in one of three ways:
+1) Automatically, using a [static import](#local-images)
+2) Explicitly, by including a `height` **and** `width` property
+3) By using the `fill` layout mode, which causes the image to expand to fill its parent element. 
 
-Images are optimized dynamically upon request and stored in the `<distDir>/cache/images` directory. The optimized image file will be served for subsequent requests until the expiration is reached. When a request is made that matches a cached but expired file, the cached file is deleted before generating a new optimized image and caching the new file.
+>### What if I don't know the size of my images?
+>
+>If you are accessing images from a source without knowledge of the images' sizes, there are several things you can do:
+>
+>**Use `layout='fill'`**
+>
+>The `fill` layout mode allows your image to be sized by its parent element. Consider using CSS to give the image's parent element space on the page, then using the [`objectFit property`](/docs/api-reference/next/image.md#objectfit) with `fill`, `contain`, or `cover`, along with the [`objectPosition property`](/docs/api-reference/next/image.md#objectposition) to define how the image should occupy that space. 
+>
+>**Normalize your images**
+>
+>If you're serving images from a source that you control, consider modifying your image pipeline to normalize the images to a specific size.
+>
+>**Modify your API calls**
+>
+>If your application is retrieving image URLs using an API call (such as to a CMS), you may be able to modify the API call to return the image dimensions along with the URL.
 
-The expiration (or rather Max Age) is defined by the upstream server's `Cache-Control` header.
+If none of the suggested methods works for sizing your images, the `next/image` component is designed to work well on a page alongside standard `<img>` elements. 
 
-If `s-maxage` is found in `Cache-Control`, it is used. If no `s-maxage` is found, then `max-age` is used. If no `max-age` is found, then [`minimumCacheTTL`](#minimum-cache-ttl) is used.
+## Styling
 
-You can configure [`minimumCacheTTL`](#minimum-cache-ttl) to increase the cache duration when the upstream image does not include `max-age`.
+Styling the image component is not that different from styling a norrmal `<img>` element, but there are a few guidelines to keep in mind:
 
-You can also configure [`deviceSizes`](#device-sizes) and [`imageSizes`](#device-sizes) to reduce the total number of possible generated images.
+**Pick the correct layout mode**
```

Comment:
This markdown style doesn't seem to work in the preview (looks like normal text). Do you need ### ?

Reply:
Hmm, which preview are you looking at? `**` renders as bold text on the Next.js app, so I'm using it as a kind of H4 heading here.

Reply:
Oh, the Github preview. Maybe it's broken?